src/FlowDiagnostics.jl

```diff
@@ -279,25 +280,16 @@ Note that EPV values are correctly calculated both in the interior and the bound
 interior and top boundary, EPV = f×N² = 10⁻¹⁰, while EPV = 0 at the bottom boundary since ∂b/∂z
 is zero there.
 """
-function ErtelPotentialVorticity(model; location = (Face, Face, Face), add_background = true)
+function ErtelPotentialVorticity(model; location = (Face, Face, Face))
     validate_location(location, "ErtelPotentialVorticity", (Face, Face, Face))
 
-    if model.buoyancy == nothing || !(model.buoyancy.formulation isa BuoyancyTracer)
-        throw(ArgumentError("`ErtelPotentialVorticity` is only implemented for `BuoyancyTracer` at the moment."))
+    if model.buoyancy == nothing
+        throw(ArgumentError("Cannot calculate `ErtelPotentialVorticity` when `model.buoyancy == nothing`."))
```

Comment:
You can actually use any tracer to compute PV (buoyancy just gives nice results because it's 1-1 with z coordinate).

Reply:
so you could let people specify the tracer which defaults to buoyancy

Reply:
Ah that's true, I forgot about that detail. I guess I don't need to transform active tracers into buoyancy...

Reply:
Buoyancy is usually the best choice so I do think it's a good idea to use it. I'm just saying without any additional work you could allow users to chose the "stretching tracer". The idea is that $\Pi_c = \bf{\omega}^a \cdot \bnabla c$ is conserved whenever $c$ is conserved.

Reply:
Yeah, 100% agree. I simply forgot that the PV definition is more general than what we end up using in practice!